src/GraphQL/Execution/ExecutionResultJsonConverter.cs

```diff
@@ -131,9 +130,8 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             throw new NotImplementedException();
         }
 
-        public override bool CanConvert(Type objectType)
-        {
-            return objectType == typeof(ExecutionResult);
-        }
+        public override bool CanRead => false;
```

Comment:
`ReadJson` throws `NotImplementedException` so it will be better explicitly return `false` instead of `true` in base class.